src/main/java/com/google/api/codegen/discovery/MainDiscoveryProviderFactory.java

```diff
@@ -85,13 +94,15 @@ public static DiscoveryProvider defaultCreate(
           .setSnippetFileName(id + "/" + DEFAULT_SNIPPET_FILE)
           .build();
     } else if (id.equals(NODEJS)) {
-      return CommonDiscoveryProvider.newBuilder()
-          .setContext(new NodeJSDiscoveryContext(service, apiaryConfig))
-          .setSnippetSetRunner(
-              new NodeJSSnippetSetRunner<Method>(SnippetSetRunner.SNIPPET_RESOURCE_ROOT))
-          .setSnippetFileName(id + "/" + DEFAULT_SNIPPET_FILE)
+      return ViewModelProvider.newBuilder()
```

Comment:
i'll remove anything like this before merging, it's just a pain to checkout this file after each update